src/clientdlg.cpp

```diff
@@ -1000,46 +999,50 @@ void CClientDlg::ShowConnectionSetupDialog()
 {
     // show connect dialog
     bConnectDlgWasShown = true;
-    ConnectDlg.show();
+    ConnectDlg.show ();
+    ConnectDlg.setWindowTitle ( MakeClientNameTitle ( tr ( "Connect" ) , pClient->strClientName ) );
 
     // make sure dialog is upfront and has focus
-    ConnectDlg.raise();
-    ConnectDlg.activateWindow();
+    ConnectDlg.raise ();
+    ConnectDlg.activateWindow ();
```

Comment:
I know the formatting style can be confusing! But when submitting a PR, there is usually no need to reformat lines that you haven't changed.

In this case, the "house style" seems to be _no_ space before an empty argument list `()`, but a space before and after an opening `(` when there are arguments to the function.